include/bpf_helpers.h

```diff
@@ -16,7 +16,44 @@
 // libbpf's bpf_helpers.h for the rest of the platform-agnostic
 // defines.
 #ifndef _MSC_VER
-#include "libbpf/src/bpf_helpers.h"
+
+// Definitions lifted from libbpf's bpf_helpers.h.
```

Comment:
Re "lifted from".
That file has
> /* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */

This one has
> // SPDX-License-Identifier: MIT

Is it ok to "lift" code from BSD-2-Clause to MIT?   (I think no, it has to have the BSD-2-Clause license identifier.)

Reply:
https://softwareengineering.stackexchange.com/questions/121998/mit-vs-bsd-vs-dual-license/ claims:

> * The MIT and the BSD 2-clause licenses are essentially identical.
**True** Although there is some ambiguity around whether some parts of the MIT license apply to binaries.

So it may be ok but check with a lawyer to be sure.

Reply:
Removed libbpf code and recreate macros from Linux Kernel documentation.